### DISCUSSION TITLE
ArcGIS Online and FGDC Symbols style

### DISCUSSION THREAD

**Discussion by moendopi** [usgs/gems-tools-pro#64] [General]
I am currently testing out an idea for a story map attached to a database I am working on. I got the database loaded into Arc Online, but when it comes to symbolizing I am running into some trouble. I want the online version, obviously, to match the database. Is there a way to load FGDC symbols into ArcGIS Online?

**Comment by ethoms-usgs** [comment]
You can [share it](https://pro.arcgis.com/en/pro-app/latest/help/sharing/overview/share-a-web-style.htm) as a web style, and that has been done for the USGS organization, but, if I understand correctly, you have to apply each symbol separately, there is no match-to-style option. 

The other thing you can do (again, second-hand knowledge) is publish the layer as part of a web map, not as a standalone hosted feature layer. The symbology will be retained in a web map, but it is downgraded in a feature layer.
